Sources/App/Models/SocialAccount/Social.swift

```diff
@@ -0,0 +1,92 @@
+import Vapor
+import FluentProvider
+import HTTP
+
+final class Social: Model {
+  let storage = Storage()
+  
+  var socialId: Int
```

Comment:
Let's discuss usage of Identifiers? @smejl @NSEGeorge

Reply:
What do you mean?

Reply:
There is a type `Identifier` in Vapor, I'm trying to understand advantages and disadvantages of it usage compared to plain types (Int, String, etc)

Reply:
I think, `Identifier` is just a universal type for all possible id, so it's easier to use when, for example, we decide to change our id type from `Int` to `String` in out DB. In code we wont change anything